src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/HdcLicence.kt

```diff
@@ -244,7 +244,7 @@ data class HdcLicence(
   override val updatedByUsername: String? = null,
 
   @Schema(description = "The curfew times for this licence")
-  val curfewTimes: HdcCurfewTimes? = null,
```

Comment:
mmh.. how is this not broken on the frontend?

Reply:
and will this change break the frontend?

Reply:
> will this change break the frontend?

I don't believe so, no. The front end expects an array of curfew times to be returned, I believe this is an existing bug that is being fixed. That said, I'm not sure how it wasn't broken before. Looking into that now.

Below is a screenshot of the HDC Curfew details in my local with my changes, it all seems to be rendering as expected to me
![image](https://github.com/user-attachments/assets/b4240e6e-27dd-46f0-b621-9781fc992980)

Reply:
magic javascript